consensus/clique/clique.go

```diff
@@ -298,6 +298,9 @@ func (c *Clique) verifyHeader(chain consensus.ChainHeaderReader, header *types.H
 	if header.GasLimit > params.MaxGasLimit {
 		return fmt.Errorf("invalid gasLimit: have %v, max %v", header.GasLimit, params.MaxGasLimit)
 	}
+	if chain.Config().IsShanghai(new(big.Int).SetUint64(header.Time)) {
```

Comment:
Hmm. Perhaps a more general question, should clique algorithm bypass all the following EIPs?
In Shanghai, there is also some other EIPs and can be activated on clique as well. Theoretically
it should be possible to ignore `The Merge` fork for clique(just like skip GrayGlacier) but still
chase the latest spec somehow.

Reply:
I think it's hard to allow parts of a fork, so probably best to just not allow any of it. Eventually we should be moving to remove clique anyways.

Reply:
Yep, we've decided in the review call to not split up the fork changes